incubator/nfs-pv-provider/templates/nfs-server-deploy.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: "{{ .Values.deployment.api_version }}"
+kind: Deployment
+metadata:
+  name: nfs-server
+  labels:
+    app: {{ template "nfs-pv-provider.name" . }}
+    chart: {{ template "nfs-pv-provider.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: nfs-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      role: nfs-server
+  template:
+    metadata:
+      labels:
+        role: nfs-server
+    spec:
+      containers:
+      - name: nfs-server
```

Comment:
Use consistent list indentation style. I'd suggest you always indent two spaces.

Reply:
> I'd suggest you always indent two spaces.

afaik, it's generally preferred in YAML that `- ` counts as indentation. Just my $0.02. Whichever way is chosen, I agree it should be applied consistently within a single YAML file.

Reply:
I was not even really conscious of this distinction and did not realize I have done it two different ways.  Addressed according to @krancour suggestion in cec82017

I will review the other files and be sure I was consistent about this everywhere.  Thanks!

Reply:
I'm seeing that the standard in my favorite Kubernetes project's charts actually appears to be two spaces before the `-` and so actually think I will go with the extra spaces instead...

Reply:
78a42910